Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM node:alpine
 
-COPY . /app
+
+RUN apk add git
```

Comment:
git should no longer be necessary.

Reply:
I was guessing it was needed for maybe a npm package that is only available via git, so I left everything the way it was. Just some restructuring..